libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -1050,4 +1067,86 @@ export const buildTigerSpecificFunctions = (
             });
         });
     },
+    getAllCSPDirectives: async (): Promise<Array<ICSPDirective>> => {
```

Comment:
Please add an empty line to separate this function from the one above.

Reply:
not sure about this. Almost functions do not have an empty line to separate each one

Reply:
ok then